components/core/src/clp_s/ArchiveWriter.cpp

```diff
@@ -49,13 +44,32 @@ size_t ArchiveWriter::close() {
     compressed_size += m_array_dict->close();
     compressed_size += m_timestamp_dict->close_local();
 
-    for (auto& i : m_schema_id_to_writer) {
-        i.second->store();
-        compressed_size += i.second->close();
+    m_table_file_writer.open(m_archive_path + "/table", FileWriter::OpenMode::CreateForWriting);
```

Comment:
Could we change the filenames for the `table` and `metadata` sections?

I think it should at least be `tables` and `table_metadata`, but we could also consider using our "official" terminology and do something like `encoded_record_tables` or `erts` and `ert_metadata`.

Reply:
Yeah, they are better